items/Project/Global/System.Workflows/Demo Workflow/Awaiting Approval/Approve/Submission Notification.yml

```diff
@@ -0,0 +1,66 @@
+﻿---
+ID: "39fe2ff1-a928-42f8-b6a5-f35187118fe9"
+Parent: "08d7311c-1cea-4db3-9466-a24721647071"
+Template: "f7a56f00-e12e-49ce-af4f-a92b2529618d"
+Path: /sitecore/system/Workflows/Demo Workflow/Awaiting Approval/Approve/Submission Notification
+DB: master
+SharedFields:
+- ID: "2320d642-d940-461d-bc55-96078daa1926"
+  Hint: From
+  Value: noreply@demo.com
+- ID: "3cb83e5d-8b67-4ba5-9bc6-e9f122ad95cb"
+  Hint: Mail Server
+  Value: smtp
+- ID: "ba3f86a2-4a1c-4d78-b63d-91c2779c1b5e"
+  Hint: __Sortorder
+  Value: 200
+Languages:
+- Language: en
+  Versions:
+  - Version: 1
+    Fields:
+    - ID: "25bed78c-4957-4165-998a-ca1b52f67497"
+      Hint: __Created
+      Value: 20180530T150228Z
+    - ID: "2c18e0dc-d31f-437d-8eb2-0b025959e57a"
+      Hint: Approvers Role
+      Value: $submitter$
+    - ID: "42dc6037-ca08-4fc8-8c70-213842d032ea"
+      Hint: Subject
+      Value: "Demo Shared: Items Approved"
+    - ID: "5dd74568-4d4b-44c1-b513-0af5f4cda34f"
+      Hint: __Created by
+      Value: |
+        sitecore\admin
+    - ID: "e8f76062-0990-4b8b-860e-de4f50eb76ea"
+      Hint: Message
+      Value: |
+        This message is generated by a demo workflow action and was sent to the user who requested for the item approval. <br />
+        The following item was approved by $actionedby$: <br /><br />
+        Comments: $comment$ <br />
+        Item path : $itemPath$ <br />
+        Item language : $itemLanguage$ <br />
+        Item Version : $itemVersion$ <br /><br />
```

Comment:
The `<br />`s at the end of this last line are not necessary. Could they please be removed?

Reply:
will do it